Cargo.toml

```diff
@@ -15,9 +15,13 @@ publish = false
 targets = ["x86_64-pc-windows-gnu", "x86_64-pc-windows-msvc"]
 
 [dependencies]
-rustls = { git = "https://github.com/rustls/rustls.git", default-features = false, features = [
-    "std",
-] }
+rustls = { version = "0.23.0", features = ["ring", "tls12", "std"], default-features = false }
+symcrypt = "0.3.0"
+rustls-platform-verifier = "0.3.4"  
+bcrypt = "0.15"
+winapi = { version = "0.3", features = ["bcrypt", "winuser"] }
```

Comment:
winapi is deprecated. Use official Microsoft windows_sys crate.

Reply:
done